financial-connections-example/src/main/java/com/stripe/android/financialconnections/example/FinancialConnectionsDataExampleActivityJava.java

```diff
@@ -0,0 +1,77 @@
+package com.stripe.android.financialconnections.example;
+
+import android.os.Bundle;
+import android.util.Log;
+import android.widget.Button;
+import android.widget.TextView;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.appcompat.widget.Toolbar;
+import androidx.lifecycle.ViewModelProvider;
+
+import com.stripe.android.financialconnections.FinancialConnections;
+import com.stripe.android.financialconnections.FinancialConnectionsSheet;
+import com.stripe.android.financialconnections.example.FinancialConnectionsExampleViewEffect.OpenFinancialConnectionsSheetExample;
+
+import kotlin.Unit;
+
+public class FinancialConnectionsDataExampleActivityJava extends AppCompatActivity {
```

Comment:
Added a java example replicating `FinancialConnectionsDataExampleActivity`. It also observes events.